src/translation.ts

```diff
@@ -0,0 +1,27 @@
+import { DocumentContent, GanomedeDataClient } from "./data-client";
```

Comment:
I think here it would be better named `localizedTemplates` or something (because it's not really translating anything).

Reply:
Rename translation and translate to localizedTemplates and localize.